packages/react-native-renderer/src/ReactFabric.js

```diff
@@ -137,6 +137,7 @@ injectIntoDevTools({
   version: ReactVersion,
   rendererPackageName: 'react-native-renderer',
   rendererConfig: {
+    getInspectorDataForInstance,
```

Comment:
Q: Why its named `getInspectorDataForInstance`?

Should it be `getInspectorDataForFiber`?